src/class/traits/object.rs

```diff
@@ -308,7 +301,7 @@ pub trait Object: From<Value> {
     ///     let mut string = RString::new_utf8("Some string");
     ///
     ///     // The same can be done by modifying `string.singleton_class()`
-    ///     // or using `string.define_singleton_method("greeting", greeting)`
+    ///     // or usiang `string.define_singleton_method("greeting", greeting)`
```

Comment:
@goyox86 Typo added

Reply:
Addressed in 0a743fb3b3d9eedbd431e17bfcdbf4e676c7c74f